### ISSUE TITLE
Pantheon Autopilot Doc Image size issue

### ISSUE THREAD

**Issue by jms-pantheon** [pantheon-systems/documentation#7358]
Re: [Pantheon Autopilot](https://pantheon.io/docs//guides/autopilot)

Priority: Low

## Issue Description:
[Site Compatibility](https://pantheon.io/docs/guides/autopilot#autopilot-site-compatibility) section has an image with text that gets scaled down to 4px. For accessibility this text should be much larger, and an image should not be the only way of getting this information or it should have accurate alt text. 

## Suggested Resolution
Replace with an alternate display

**Comment by jms-pantheon** [comment]
Thanks @whitneymeredith !

**Comment by whitneymeredith** [comment]
Thanks for bringing it to our attention @jms-pantheon !
